src/bundle/Resources/views/content/tabs/versions/table.html.twig

```diff
@@ -41,9 +41,13 @@
             <td>
                 {{ version.modificationDate|localizeddate('medium', 'short', app.request.locale) }}
             </td>
-            {% if isDraft is defined %}
+            {% if isDraft is defined and version.canEdit %}
```

Comment:
the `version.canEdit` check should be separated affect only the generation (or not) of the link, otherwise if the user can not edit a version, the cell will be missing in the table.